build.py

```diff
@@ -0,0 +1,357 @@
+        return not (path in excluded_paths or (windows and path.lower() in excluded_paths))
+    return search_path(name, inclusion_predicate=should_include)
+
+def proc_call(argv, **kwargs):
+    # The 'check' keyword argument can be a a bool, to indicate if the output should be checked. If False, then it is returned as part of the output.
+    # The 'stdin' keyword argument can be a string, in which case it is communicated to the process. If stdin is a string but stdout is unspecified, stdout is piped to a string.
+    # The 'stdout' keyword argument can be subprocess.PIPE, in which case it is returned. Do NOT forget to rstrip() it if you need ot avoid trailing newlines!
+    check = kwargs.pop('check', True)
+    stdin = kwargs.get('stdin')
+    rstrip = kwargs.pop('rstrip', True)
+    if isinstance(stdin, StringishTypes):
+        kwargs['stdin'] = subprocess.PIPE
+        kwargs.setdefault('stdout', subprocess.PIPE)
+    else:
+        stdin = None
+    full_command = make_command_line(argv)
+    proc = subprocess.Popen(full_command, **kwargs)
+    retcode = proc.wait()
+    if check and retcode: raise subprocess.CalledProcessError(retcode, full_command[0] if not isinstance(full_command, StringishTypes) else full_command)
+    results = list(proc.communicate(stdin))
+    if sys.version_info[0] >= 3 and 'encoding' not in kwargs and isinstance(stdin, StringishTypes):
+        for r, result in enumerate(results):
+            if isinstance(stdin, str) and isinstance(result, bytes):
+                results[r] = result = result.decode('utf-8')
+    if rstrip:
+        for r, result in enumerate(results):
+            if isinstance(result, StringishTypes):
+                if len(result) > 0 and result[-1:] in (u'\n', b'\n'):
+                    results[r] = result = result[:-1]
+                if len(result) > 0 and result[-1:] in (u'\r', b'\r'):
+                    results[r] = result = result[:-1]
+    return (retcode,) + tuple(results)
+
+def shell_call(shell, *commands, **kwargs):
+    sep = kwargs.pop('sep', '&&')  # Command separator. If explicitly passed as None, strings can be interspersed between commands (which are themselves lists of strings).
+    return proc_call([shell, '-c'] + make_bash_command_line('&&' if sep is None else sep, *commands, unsafe=sep is None), **kwargs)
+
+def make_shell_caller(*bound_args, **bound_kwargs):
+    return lambda *args, **kwargs: shell_call(*(bound_args + args), **dict(bound_kwargs, **kwargs))
+
+def get_current_python_cli_executable():
+    result = sys.executable
+    if sys.platform == 'win32':
+        # On Windows, CLI vs. GUI (Windowed) Python have different names; the Window versions end with 'w'
+        (dirname, basename) = os.path.split(result)
+        (name, ext) = os.path.splitext(basename)
+        result = os.path.join(dirname, name.rstrip('w').rstrip('W') + ext)
+    return result
+
+def call_bazel(bazel, *args, **kwargs):  # SINGLE-THREADED CALLS ONLY
```

Comment:
We shoudn't have this. If people want to configure bazel, they should do it via the `.bazelrc`.

Reply:
Confused, isn't the build script supposed to call bazel?